_sass/_base.scss

```diff
@@ -1,204 +1,204 @@
 /**
- * Reset some basic elements
- */
+* Reset some basic elements
+*/
 body, h1, h2, h3, h4, h5, h6,
 p, blockquote, pre, hr,
 dl, dd, ol, ul, figure {
-    margin: 0;
-    padding: 0;
+  margin: 0;
+  padding: 0;
 }
 
 
 
 /**
- * Basic styling
- */
+* Basic styling
+*/
 body {
-    font-family: $base-font-family;
-    font-size: $base-font-size;
-    line-height: $base-line-height;
-    font-weight: 300;
-    color: $text-color;
-    background-color: $background-color;
-    -webkit-text-size-adjust: 100%;
+  font-family: $base-font-family;
```

Comment:
Properties should be ordered background-color, color, font-family, font-size, font-weight, line-height, -webkit-text-size-adjust<br>Each selector in a comma sequence should be on its own line